[16.0][ADD] edi_partner_oca

QuocDuong1306 opened the pull request.

Commit:
[ADD] edi_partner_oca

tuantrantg commented on edi_partner_oca/README.rst:
```diff
@@ -0,0 +1,72 @@
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
```
@QuocDuong1306 oi, please add [CREDITS.rst](https://github.com/QuocDuong1306/edi/pull/9/files#diff-22feadc940d8006276dd166eda0a3fb73da8a35175d7a27a31e3fdc4adb33ee1) and re-generate README.rst

QuocDuong1306 replied:
Thank a Bond, I updated

nilshamerlinck commented on edi_partner_oca/__manifest__.py:
```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 ForgeFlow S.L. (http://www.forgeflow.com)
```
bro

nilshamerlinck replied:
```
# Copyright 2024 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
# Copyright 2024 Camptocamp SA
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

nilshamerlinck replied:
note the 2024

QuocDuong1306 replied:
Thank @nilshamerlinck , I updated

Commit:
[IMP] edi_product_oca: Plug EDI framework to product.packaging

Commit:
[IMP] edi_oca: add partner form page

Commit:
[ADD] edi_product_multi_barcode_oca

OCA-git-bot committed:
Merge PR #50 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update edi_product_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #54 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update edi_product_multi_barcode_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #52 into 16.0

Signed-off-by etobella

Commit:
[UPD] Update edi_oca.pot

Commit:
[UPD] Update edi_sale_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: edi-framework-16.0/edi-framework-16.0-edi_oca
Translate-URL: https://translation.odoo-community.org/projects/edi-framework-16-0/edi-framework-16-0-edi_oca/

nilshamerlinck committed:
[FIX] edi_oca: tests using FakeModelLoader should not be post_install

OCA-git-bot committed:
Merge PR #55 into 16.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates
